tests/shared/UserLogonRequestTestDefs.js

```diff
@@ -0,0 +1,517 @@
+                // for the following request to send.
+                configName: "gpii.tests.acceptance.userLogon.config",
+                configPath: "%gpii-universal/tests/configs"
+            },
+            gradeNames: ["gpii.tests.userLogonRequest.testCaseHolder", "gpii.test.integration.testCaseHolder.linux"],
+            gpiiKey: testDefs.gpiiKey || gpii.tests.userLogonRequest.gpiiKey
+        }, testDef);
+    });
+};
+
+
+gpii.tests.userLogonRequest.testDefs = [{
+    name: "Testing standard proximityTriggered login and logout",
+    expect: 29,
+    sequence: [{
+        // The initial active GPII key is "noUser"
+        func: "gpii.tests.userLogonRequest.verifyActiveGpiiKey",
+        args: ["{lifecycleManager}", ["noUser"]]
+    }, {
+        // 1. 1nd proximityTriggered request to key in testUser1
+        func: "gpii.tests.invokePromiseProducer",
+        args: ["{lifecycleManager}.performProximityTriggered", [gpii.tests.userLogonRequest.gpiiKey], "{that}"]
+    }, {
+        changeEvent: "{lifecycleManager}.applier.modelChanged",
+        path: "logonChange",
+        listener: "gpii.tests.userLogonRequest.modelChangeChecker",
+        args: ["{arguments}.0", "logout", true, "noUser"]
+    }, {
+        changeEvent: "{lifecycleManager}.applier.modelChanged",
+        path: "logonChange",
+        listener: "gpii.tests.userLogonRequest.modelChangeChecker",
+        args: ["{arguments}.0", "logout", false, "noUser"]
+    }, {
+        changeEvent: "{lifecycleManager}.applier.modelChanged",
+        path: "logonChange",
+        listener: "gpii.tests.userLogonRequest.modelChangeChecker",
+        args: ["{arguments}.0", "login", true, "testUser1"]
+    }, {
+        changeEvent: "{lifecycleManager}.applier.modelChanged",
+        path: "logonChange",
+        listener: "gpii.tests.userLogonRequest.modelChangeChecker",
+        args: ["{arguments}.0", "login", false, "testUser1"]
+    }, {
+        event: "{that}.events.onResponse",
+        listener: "gpii.tests.userLogonRequest.testLoginResponse",
+        args: ["{arguments}.0", gpii.tests.userLogonRequest.gpiiKey]
+    }, {
+        func: "gpii.tests.userLogonRequest.verifyActiveGpiiKey",
+        args: ["{lifecycleManager}", [gpii.tests.userLogonRequest.gpiiKey]]
+    }, {
+        // 2. wait for the debounce period to pass so that the following proximityTriggered request is not rejected
+        func: "setTimeout",
+        args: [ "{tests}.events.timeoutComplete.fire", 3500 ]
+    }, {
+        event: "{tests}.events.timeoutComplete",
+        listener: "fluid.identity"
+    }, {
+        // 3. 2nd proximityTriggered request to key out testUser1
+        func: "gpii.tests.invokePromiseProducer",
+        args: ["{lifecycleManager}.performProximityTriggered", [gpii.tests.userLogonRequest.gpiiKey], "{that}"]
+    }, {
+        changeEvent: "{lifecycleManager}.applier.modelChanged",
```

Comment:
FIVE passive fixtures in a row! I really wonder how this works. Perhaps it is helped by each of the actions being asynchronous as a result of your timeout.